tests/integration/examples/test_iteration_with_cluster.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from adcm_aio_client import ADCMSession, Credentials
+from tests.integration.setup_environment import ADCMContainer
+
+pytestmark = [pytest.mark.asyncio]
+
+
+async def test_iteration_with_cluster(adcm_for_examples: ADCMContainer) -> None:
+    """
+    Interaction with clusters: creating, deleting, getting a list of clusters using filtering,
+    configuring cluster configuration, launching actions on the cluster and updating the cluster.
+    """
+    credentials = Credentials(username="admin", password="admin")  # noqa: S106
+    url = adcm_for_examples.url
+    kwargs: dict = {
```

Comment:
IMO it'll be reasonable to extract "default"/"reasonable" timeout keys to a constant

Same about credentials

Reply:
I added constants for `timeout`, `retry_attempts` and `retry_interval`, but I left the `credentials` as it is, so that users could see exactly how to transfer accesses without moving to another file to search for an example.